roles/postgres/tasks/main.yml

```diff
@@ -54,39 +54,47 @@
     state: started
     enabled: true
 
-- name: Configure Alfresco related Databases
+- name: Configure Alfresco databases
   become: true
   become_user: postgres
   vars:
     ansible_ssh_pipelining: yes
   block:
-    - name: "Create a new database with name {{ item }}"
+    - name: "Create necessary databases"
       postgresql_db:
         name: "{{ item }}"
       loop:
         - "{{ repo_db_name }}"
         - "{{ sync_db_name }}"
     
-    - name: "Connect to {{ repo_db_name }} database and set user's password with no expire date"
+    - name: "Give {{ repo_db_username }} user to access {{ repo_db_name }} database with password"
       postgresql_user:
         db: "{{ repo_db_name }}"
         name: "{{ repo_db_username }}"
         password: "{{ repo_db_password }}"
         priv: "ALL"
         expires: infinity
         role_attr_flags: SUPERUSER
-      no_log: true
```

Comment:
How do we ensure running ansible in debug doesn't expose sensitive data then?

Reply:
since it's an upstream module I hope they are taking care of it - looks like so https://github.com/ansible-collections/community.postgresql/blob/218641f24473574c3d59cad2b5bb5b9ad886982f/plugins/modules/postgresql_user.py#L905